### PR TITLE
MBS-9632: Convert Stats to React (VII)

### PR DESCRIPTION
**Pull request by reosarevok**
Edits

### CODE REVIEW

**Comment by reosarevok** [comment]
We can choose to have 
`import {l as l_mb_server} from '../static/scripts/common/i18n';` (more consistent, but makes eslint cry) or `import {l as lMbServer} from '../static/scripts/common/i18n';` (camel case, but these functions usually aren't). Which is preferred?

File: root/statistics/Edits.js [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+
+type EditsStatsT = {|
+  +$c: CatalystContextT,
+  +dateCollected: string,
+  +stats: {[string]: number},
+  +statsByCategory: {[string]: $ReadOnlyArray<EditCategoryT>},
+|};
+
+const Edits = ({$c, dateCollected, stats, statsByCategory}: EditsStatsT) => {
+  console.log(stats);
```
Comment (mwiencek) [inline root/statistics/Edits.js]: ❗️

File: root/statistics/Edits.js [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+  return (
+    <StatisticsLayout fullWidth page="edits" title={l('Edits')}>
+      {manifest.css('statistics')}
```
Comment (mwiencek) [inline root/statistics/Edits.js]: `StatisticsLayout` already includes the CSS, so you can remove this plus the `manifest` import at the top (I'll have to do it in my Webpack PR in any case)

File: root/statistics/Edits.js [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+      <h2>{l('Edits')}</h2>
+      {Object.keys(statsByCategory).length === 0 ? (
+        <p>
+          {l('No edit statistics available.')}
+        </p>
+      ) : (
+        <table className="database-statistics">
+          <tbody>
+            <tr>
+              <th colSpan="2">{l('Edits:')}</th>
```
Comment (mwiencek) [inline root/statistics/Edits.js]: `addColon` here?

File: root/statistics/Edits.js (hunk 2 above)
Comment (reosarevok) [reply root/statistics/Edits.js]: Did that for every file in a second commit. One of the files (root/statistics/LanguagesScripts.js) also includes {manifest.js('statistics')}, which is not in the Layout, so I left that in.

**Pull request merged**